bolt-design-system/bolt
# Update Drupal Lab + Vue Example Integrations

## Jira
N/A

## Summary
Updates Bolt's example integrations with Drupal Lab + with Vue to use the latest Bolt / NPM dependencies + fix minor issues. Should also fix the error reported in https://github.com/bolt-design-system/bolt/issues/1092

## How to test
1. Confirm the Drupal Lab example in the `example-integrations/drupal-lab` installs + runs as expected: `composer run setup` + `composer run serve`
2. Also confirm that the Vue.js example in the `example-integrations/vue` folder installs + runs locally without throwing any errors (`yarn` + `npm start`)
